deploy-manage/deploy/elastic-cloud/azure-native-isv-service.md

```diff
@@ -254,6 +255,9 @@ $$$azure-integration-migrate$$$How do I migrate my data from the classic Azure m
     8. Confirm the data has moved successfully into your new {{es}} resource on Azure.
     9. To remove the old Azure subscription and the old deployments, go to the [Azure SaaS page](https://portal.azure.com/#blade/HubsExtension/BrowseResourceBlade/resourceType/Microsoft.SaaS%2Fresources) and unsubscribe from the {{ecloud}} ({{es}}) marketplace subscription. This action triggers the existing deployments termination.
 
+$$$azure-integration-multiple-users$$$Can multiple Azure users deploy the the same {{ecloud}} Elastic Cloud organization?
+:   Yes! Before another user creates a native resource from the Azure Portal, invite them to your {{ecloud}} Organization at https://cloud.elastic.co/account/members. When they create the resource, it will get added to the existing Organization instead of creating a new one, and allow you to benefit from consolidated billing, RBAC, and other benefits of an {{ecloud}} organization.
```

Comment:
```suggestion
:   Yes! Before another user creates a native resource from the Azure Portal, invite them to your {{ecloud}} organization at https://cloud.elastic.co/account/members. When they create the resource, it will get added to the existing organization instead of creating a new one, and allow you to benefit from consolidated billing, RBAC, and other benefits of an {{ecloud}} organization.
```